api/src/pcapi/routes/backoffice_v3/templates/accounts/get/general.html

```diff
@@ -100,6 +102,7 @@ <h6 class="card-subtitle text-muted">User ID : {{ user.id }}</h6>
                 <div class="d-flex align-items-center">
                   <span class="fw-bold me-1">Email :</span> {{ user.email }}
                   {% if user.email_history | length > 0 %}<i class="bi bi-bag-check-fill text-warning ms-1 fs-4 pc-email-changed-icon"></i>{% endif %}
+                  <span class="ms-1">{{ clipboard.copy_to_clipboard(user.email, "Copier adresse email.") }}</span>
```

Comment:
```suggestion
                  <span class="ms-1">{{ clipboard.copy_to_clipboard(user.email, "Copier l'adresse email.") }}</span>
```
+ (anecdotique) je ne suis pas convaincu par les points en fin de _tooltip_, ce n'est pas vraiment une phrase